specs/0116-LLM-assisted-translations.md

```diff
@@ -0,0 +1,34 @@
+- Feature Name: LLM-Assisted Translations in Pontoon
+- Created: 2024-01-12
+- Associated Issue: #3064
+
+# Summary
+
+Introduce LLM-assisted translations in Pontoon to provide alternative, formal, and informal translation options, enhancing the localization process and community health.
+
+# Motivation
+
+In the realm of machine translation, recent studies have highlighted the evolving capabilities of Large Language Models (LLMs). This feature aims to revitalize community engagement and improve the efficiency of localization in Mozilla's projects. Incorporating LLM-assisted translations keeps Mozilla competitive with other TMS providers, adopting new technology for better community health and more effective translation.
+
+# Feature explanation
+
+In the Machinery tab of the Pontoon translate view, users will encounter a new dropdown option for each 100% translation memory match for a given string, offering three choices: 
```

Comment:
Is it "each 100% translation memory match" or "each machine translation suggestion" or both?

Reply:
Each machine translation suggestion